ads/google/adsense.js

```diff
@@ -41,6 +41,12 @@ export function adsense(global, data) {
   if (data['adHost']) {
     i.setAttribute('data-ad-host', data['adHost']);
   }
+  if (data['adtest']) {
+    i.setAttribute('data-adtest', data['adtest']);
+  }
+  if (data['tagOrigin']) {
+    i.setAttribute('data-tag-origin', data['tagOrigin']);
+  }
```

Comment:
As these attributes accumulate, would it be worth factoring this away from a bunch of individual 'if' statements and to a foreach over a list of input/output name pairs?